src/free_functions.rs

```diff
@@ -88,16 +88,14 @@ pub fn aview1<A>(xs: &[A]) -> ArrayView1<A> {
 }
 
 /// Create a two-dimensional array view with elements borrowing `xs`.
+///
+/// **Panics** if the product of non-zero axis lengths or the numer of bytes in
```

Comment:
I don't see why this would be in bytes, actually. "numer" → "number".

Reply:
With the assumption that slices cannot contain more than `isize::MAX` bytes, the "numer of bytes" condition can be removed. Good catch on the typo :).